omod/src/main/java/org/openmrs/module/registrationapp/fragment/controller/field/DropDownFragmentController.java

```diff
@@ -25,7 +25,7 @@ public void controller(FragmentConfiguration config) throws Exception {
             Concept conceptSet = Context.getConceptService().getConceptByUuid(conceptSetUuid);
 
             for (Concept concept: conceptSet.getSetMembers()) {
-            	Map < String, Object > option = new HashMap < String, Object > ();
+            	Map < String, Object > option;
                 option = new HashMap < String, Object > ();
```

Comment:
```suggestion
                Map <String, Object> option = new HashMap <> ();
```
What do you think about such a solution?

Also correct the link to the ticket, because it contains an error. The correct one is: https://issues.openmrs.org/browse/RA-1803

Reply:
I think this looks more clear and straight.
Thanks for the suggestion.